### PR TITLE
some reworking of intro

### PR DESCRIPTION
**Pull request by asiepel**
some reorganization and editing of intro, as discussed on issues

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -229,8 +198,10 @@ \section*{Introduction}
 either a command line interface or a simple Python API, users can specify which
 organism, genetic map, chromosome, and demographic history they are interested in simulating, and the
 simulation output from their chosen model is returned.
+In this way, \stdpopsim will enable rigorous evaluation of population genetic software,
```
Comment (andrewkern) [inline popSimManu.tex:201]: do we need to define what stdpopsim is before this point?

**Comment by asiepel** [comment]
it is defined earlier in the paragraph

On Dec 20, 2019, at 12:31 AM, Andrew Kern <notifications@github.com<mailto:notifications@github.com>> wrote:


@andrewkern commented on this pull request.

________________________________

In popSimManu.tex<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_popsim-2Dconsortium_manuscript_pull_119-23discussion-5Fr360232913&d=DwMCaQ&c=mkpgQs82XaCKIwNV8b32dmVOmERqJe4bBOtF0CetP9Y&r=rNAqNCgksGfltoLkih1npA&m=0iDfnf76T7z3b67TG-506T65YHEXUY0dtpEmTOIVhOM&s=6Kdh8aYk34VcLAmH7mI1FFDJg_r6Ut15xVtyNoAho0M&e=>:

> @@ -229,8 +198,10 @@ \section*{Introduction}
 either a command line interface or a simple Python API, users can specify which
 organism, genetic map, chromosome, and demographic history they are interested in simulating, and the
 simulation output from their chosen model is returned.
+In this way, \stdpopsim will enable rigorous evaluation of population genetic software,


do we need to define what stdpopsim is before this point?

File: popSimManu.tex [hunk 2]
```diff
@@ -229,8 +198,10 @@ \section*{Introduction}
...
+and contribute to increased reliability of population genetic inferences.
 
-The software has been developed by the PopSim Consortium using a
+The \stdpopsim software has been developed by the PopSim Consortium using a
```
Comment (andrewkern) [inline popSimManu.tex:204]: "\stdpopsim library"?

**Comment by asiepel** [comment]
yes that’s better.  I’ll change

On Dec 20, 2019, at 12:33 AM, Andrew Kern <notifications@github.com<mailto:notifications@github.com>> wrote:


@andrewkern commented on this pull request.

________________________________

In popSimManu.tex<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_popsim-2Dconsortium_manuscript_pull_119-23discussion-5Fr360233186&d=DwMCaQ&c=mkpgQs82XaCKIwNV8b32dmVOmERqJe4bBOtF0CetP9Y&r=rNAqNCgksGfltoLkih1npA&m=NAH9Yy0_HRsRcsAfZzDaj4YaE4PkYgYt2kRcoIcO5AQ&s=4jrTa9_nsvmfpHdomh0stFWET3VSPrw8M9-A7amCkqM&e=>:

>
-The software has been developed by the PopSim Consortium using a
+The \stdpopsim software has been developed by the PopSim Consortium using a


"\stdpopsim library"?

**Review by andrewkern — COMMENTED** [review]
I like it but left a couple comments in here. What should we do about introducing "stdpopsim" the first time?

**Comment by asiepel** [comment]
I actually think it’s okay, no?  It’s introduced in the penultimate paragraph.   Name isn’t used until that point.

On Dec 20, 2019, at 12:34 AM, Andrew Kern <notifications@github.com<mailto:notifications@github.com>> wrote:


@andrewkern commented on this pull request.

I like it but left a couple comments in here. What should we do about introducing "stdpopsim" the first time?

**Comment by andrewkern** [comment]
yeah i think it is okay on second glace. don't close this PR btw-- just push more changes to your branch and then will show up here

**Comment by asiepel** [comment]
ah ok

On Dec 20, 2019, at 12:37 AM, Andrew Kern <notifications@github.com<mailto:notifications@github.com>> wrote:


yeah i think it is okay on second glace. don't close this PR btw-- just push more changes to your branch and then will show up here

**Pull request closed without merging**